psd-web/Gemfile

```
@@ -42,7 +42,7 @@ gem "strong_migrations", "0.6.2"
 gem "webpacker", "4.2.2"
 gem "wicked"
 
-gem "govuk-design-system-rails", git: "https://github.com/UKGovernmentBEIS/govuk-design-system-rails", tag: "0.6.2", require: "govuk_design_system"
+gem "govuk-design-system-rails", git: "https://github.com/UKGovernmentBEIS/govuk-design-system-rails", tag: "0.6.3", require: "govuk_design_system"
```

Exactly the same but slightly shorter:
```suggestion
gem "govuk-design-system-rails", github: "UKGovernmentBEIS/govuk-design-system-rails", tag: "0.6.3", require: "govuk_design_system"
```

---

This was the original form but I think rubocop complained about it.